docs/sql-ref-syntax-ddl-truncate-table.md

```diff
@@ -52,47 +55,44 @@ TRUNCATE TABLE table_identifier [ partition_spec ]
   </dd>
 </dl>
 
-
 ### Examples
-{% highlight sql %}
 
---Create table Student with partition
-CREATE TABLE Student ( name String, rollno INT) PARTITIONED BY (age int);
+{% highlight sql %}
+-- Create table Student with partition
+CREATE TABLE Student (name STRING, rollno INT) PARTITIONED BY (age INT);
 
 SELECT * from Student;
-+-------+---------+------+--+
-| name  | rollno  | age  |
-+-------+---------+------+--+
-| ABC   | 1       | 10   |
-| DEF   | 2       | 10   |
-| XYZ   | 3       | 12   |
-+-------+---------+------+--+
+  +----+------+---+
+  |name|rollno|age|
+  +----+------+---+
+  | ABC|     1| 10|
+  | DEF|     2| 10|
+  | XYZ|     3| 12|
+  +----+------+---+
 
 -- Removes all rows from the table in the partition specified
 TRUNCATE TABLE Student partition(age=10);
 
---After truncate execution, records belonging to partition age=10 are removed
+-- After truncate execution, records belonging to partition age=10 are removed
 SELECT * from Student;
-+-------+---------+------+--+
-| name  | rollno  | age  |
-+-------+---------+------+--+
-| XYZ   | 3       | 12   |
-+-------+---------+------+--+
+  +----+------+---+
+  |name|rollno|age|
+  +----+------+---+
+  | XYZ|     3| 12|
+  +----+------+---+
 
 -- Removes all rows from the table from all partitions
 TRUNCATE TABLE Student;
 
 SELECT * from Student;
-+-------+---------+------+--+
-| name  | rollno  | age  |
-+-------+---------+------+--+
-+-------+---------+------+--+
-No rows selected 
-
+  +----+------+---+
+  |name|rollno|age|
+  +----+------+---+
+  +----+------+---+
+  No rows selected 
```

Comment:
nit: This line is a comment, right? put ```--``` in front of it?

Reply:
Ah, yes. Probably, we can remove this. Thanks!